### PR TITLE
WIP: Add Mentor Graphics parser tool

### PR DESCRIPTION
**Pull request by derrickgw**

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/analysis/warnings/MentorGraphics.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import org.kohsuke.stapler.DataBoundConstructor;
+
+/**
+ * Provides a parser and customized messages for the Cadence Incisive Enterprise Simulator.
+ *
+ * @author Derrick Gibelyou
+ */
+public class MentorGraphics extends ReportScanningTool {
+    private static final long serialVersionUID = 8284958840616127492L;
+    private static final String ID = "cadence";
```
Comment (uhafner) [inline src/main/java/io/jenkins/plugins/analysis/warnings/MentorGraphics.java]: CP typo: `mentor`

File: src/main/java/io/jenkins/plugins/analysis/warnings/MentorGraphics.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+import edu.hm.hafner.analysis.parser.MentorParser;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import io.jenkins.plugins.analysis.core.model.ReportScanningTool;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+/**
+ * Provides a parser and customized messages for the Cadence Incisive Enterprise Simulator.
```
Comment (uhafner) [inline src/main/java/io/jenkins/plugins/analysis/warnings/MentorGraphics.java]: Update with your tool name

File: src/main/java/io/jenkins/plugins/analysis/warnings/MentorGraphics.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+         */
+        public Descriptor() {
+            super(ID);
+        }
+
+        @NonNull
+        @Override
+        public String getDisplayName() {
+            return Messages.Warnings_MentorGraphics_ParserName();
+        }
```
Comment (uhafner) [inline src/main/java/io/jenkins/plugins/analysis/warnings/MentorGraphics.java]: You can add some more properties if that makes sense for your tool (help, default pattern, icons)

**Comment by derrickgw** [comment]
Do I need to bump the minor version since I added a class?

**Comment by uhafner** [comment]
Yes, since we add a feature, the next version will be 5.1.0-SNAPSHOT.

**Comment by derrickgw** [comment]
Do I edit that manually, or is there a maven command to do that?

**Comment by uhafner** [comment]
You need to change the version in the pom.xml manually to 5.1.0-SNAPSHOT.

**Pull request merged**